site2/website/blog/2021-02-03-Five-Minute-Interview-Elastic.md

```diff
@@ -0,0 +1,45 @@
+
+### How did you decide that Pulsar was the right technology to adopt? Were there specific features you wanted to take advantage of?
+
+I don't really believe in this concept of "right technology" without a proper context. Developers in need of a streaming data technology that may be evaluating Pulsar as an option should come up with their own reasoning. I believe that Pulsar has some unique capabilities that if relevant in the right context may make Pulsar look good when compared to others.
+
+I personally love the Pulsar APIs and how simple they are in terms of making the coding simpler and fun. The fact that Pulsar is both a streaming and a messaging technology also pleases me. It is not uncommon to find scenarios where you need both in the same application and Pulsar delivers it beautifully. Arguably this is what "simplification of IT" should be about.
+
+The storage system of Pulsar also solves the very critical problem of handling more load without necessarily adding more compute capacity. You can easily increase just the storage which is cheaper.
+
+Finally, Pulsar provides a more simpler ecosystem to manage. Most of the components that are necessary in a real data pipeline application (such as schema registries, connectors, transformers) are available OOTB. While physically they are still different instances of a given deployment, they are not necessarily foreign technologies. Ultimately everything is all Pulsar brokers with different roles.
+
+### What was the most useful resource or content to you as you got started?
+
+The Apache Pulsar website and its documentation was pretty much all I had to read in order to get started with the technology and getting things done. Occasionally I had to visit the project's [source code on GitHub](https://github.com/apache/pulsar) to understand certain behaviors (such as how to proper parse a topic name to get info about tenant, namespace, etc) and also the [community available on Slack](https://pulsar.apache.org/contact/). People there are friendly and knowledgeable.
+
+### What advice would you have for other organizations who are considering Pulsar?
+
+I would say, "try to remember that Pulsar is a distributed system". While this may sound obvious at first, it is incredible how many people forget about this and try to adopt certain technologies without putting this in check. Distributed systems are inherently complex to manage and though Pulsar simplifies a lot the operations side of things it is still a complex beast that needs to be managed — which makes things harder if you don't have (or don't want to have) this skill in-house. Luckily we have managed services that can alleviate a lot of this pain and if you're a company that is not in the technology business — you better take managed services for granted.
```

Comment:
```suggestion
I would say, "try to remember that Pulsar is a distributed system". While this may sound obvious at first, it is incredible how many people forget about this and try to adopt certain technologies without putting this in check. Distributed systems are inherently complex to manage and though Pulsar simplifies a lot the operations side of things, it is still a complex beast that needs to be managed — which makes things harder if you don't have (or don't want to have) this skill in-house. Luckily, we have managed services that can alleviate a lot of this pain and if you're a company that is not in the technology business — you'd better take managed services for granted.
```